test/mount/test_mount.rs

```diff
@@ -89,6 +91,8 @@ fn test_mount_rdonly_disallows_write() {
             .unwrap()
     );
 
+    // wait for child processes to prevent EBUSY
+    let _m = FORK_MTX.lock();
```

Comment:
Is this lock necessary, this test does not open file descriptors, so theoretically no one could access the file system mounted at `tempdir`

Reply:
You are right, the `File::create` has to fail, so this is not necessary